src/Symfony/Component/Translation/Loader/CsvFileLoader.php

```diff
@@ -38,7 +38,7 @@ protected function loadResource(string $resource)
         }
 
         $file->setFlags(\SplFileObject::READ_CSV | \SplFileObject::SKIP_EMPTY);
-        $file->setCsvControl($this->delimiter, $this->enclosure, $this->escape);
+        $file->setCsvControl($this->delimiter, $this->enclosure, \PHP_VERSION_ID >= 70400 ? '' : $this->escape);
```

Comment:
I think rather need an approach similar to what we did in #32051 to still allow developers to use custom escape characters using `setCsvControl()`.

Reply:
Is this new version better and matches what you have in mind?

Reply:
yes, looks good to me 👍